library/src/main/java/com/prolificinteractive/materialcalendarview/CalendarDay.java

```diff
@@ -12,20 +13,78 @@
  */
 public final class CalendarDay implements Parcelable {
 
+    /**
+     * @return CalendarDay set to today's date
+     */
+    public static @NonNull CalendarDay today() {
+        return from(CalendarUtils.getInstance());
+    }
+
+    /**
+     *
+     * @param year new instance's year
+     * @param month new instance's month as defined by {@linkplain java.util.Calendar}
+     * @param day new instance's day of month
+     * @return CalendarDay set to the specified date
+     */
+    public static @NonNull CalendarDay from(int year, int month, int day) {
+        return new CalendarDay(year, month, day);
+    }
+
+    /**
+     * @return CalendarDay set to the specified date
+     */
+    public static CalendarDay from(Calendar calendar) {
```

Comment:
@chahinem I added nullable annotation to the parameter but not the return type, since it's an unlikely case. It will only be null if you pass null.